packages/components/src/DataViewer/Core/Tree/Tree.scss

```diff
@@ -14,6 +14,7 @@
 
 	&-node-border {
 		border-left: 1px solid $alto;
+		margin-left: -12px;
```

Comment:
Why 12px ?

Reply:
It is to align the border below the branch icon, right in the middle.